### PR TITLE
Fix preview test

### PR DESCRIPTION
**Pull request by kyrias**
Fixes test issues raised in #1664.

### CODE REVIEW

File: tests/test_preview.py [hunk 1]
```diff
@@ -150,13 +150,13 @@ def test_simple(self):
         Some text.
         </body>
         </html>
-        """
+        u"""
```
Comment (richvdh) [inline tests/test_preview.py]: think this has got lost. also a few below.

**Comment by richvdh** [comment]
matrixbot: ok to test

File: tests/test_preview.py (hunk 1 above)
Comment (kyrias) [inline tests/test_preview.py]: Gah, d'oh, to greedy of a regex, sorry about that, fixed now.

**Comment by richvdh** [comment]
Thanks!

**Pull request merged**